lib/usd/translators/mayaReferenceUpdater.cpp

```diff
@@ -45,7 +45,11 @@ PXRUSDMAYA_REGISTER_UPDATER(
     reference,
     PxrUsdTranslators_MayaReferenceUpdater,
     (UsdMayaPrimUpdater::Supports::Push | UsdMayaPrimUpdater::Supports::Clear
-     | UsdMayaPrimUpdater::Supports::AutoPull));
+     // Disable AutoPull pending more granular support for it: MayaReference
```

Comment:
Change is out of scope for this pull request: auto pull is not based on type, it's based on an instance of a type, i.e. because we need to ask the instance whether it can auto pull or not (and an instance without a file path should not be auto pulled).  We will need a virtual function to customize this.